PhysicsTools/HeppyCore/scripts/heppy_batch.py

```diff
@@ -63,7 +63,7 @@ def batchScriptCERN( jobDir, remoteDir=''):
 do
    ff=`basename $f | cut -d . -f 1`
    #d=`echo $f | cut -d / -f 2`
-   gfal-mkdir -p {srm}
+   gfal-mkdir {srm}
```

Comment:
I think the version with -p may be the more correct one ( @brucala please confirm )
if yes, can you please add a commit to put back the -p ?